skeleton/aws/modules/vpc/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "namespace" {
+}
+
+variable "cidr" {
+  description = "VPC CIDR"
+  default     = "10.0.0.0/16"
+}
+
+variable "private_subnet_cidrs" {
```

Comment:
```suggestion
variable "private_subnet_cidrs" {
  description = "VPC private subnet CIDRs"
```

Reply:
Added in https://github.com/nimblehq/infrastructure-templates/pull/27/commits/fe3318a48516b380ea6b34a5347ede9d6a35cfa0